app/views/consolidated_mailer/course/_assessment_opening.html.slim

```diff
@@ -0,0 +1,4 @@
+= t('.section_header')
```

Comment:
I'm not sure if we want to cater for the possibility of using the tab names or section names or component names (if the component name is changed).